pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -484,8 +484,160 @@ def test_parse_latex_css_conversion(css, expected):
     assert result == expected
 
 
+@pytest.mark.parametrize("environment", ["tabular", "longtable"])
+def test_parse_latex_css_convert_minimal(styler, environment):
+    # parameters ensure longtable template is also tested
+    styler.highlight_max(props="font-weight:bold;")
+    result = styler.to_latex(convert_css=True, environment=environment)
+    assert (
+        dedent(
+            f"""\
+        0 & 0 & \\bfseries -0.61 & ab \\\\
+        1 & \\bfseries 1 & -1.22 & \\bfseries cd \\\\
+        \\end{{{environment}}}
+    """
```

Comment:
Is this indentation of the closing triple quotes right?

Reply:
yes, if you have an 'f' string with a variable `var='text'` and the result should be '{text}' you need f"{{{var}}}"

Reply:
I was referring to the triple quotes indented differently, not to the curly braces.

Reply:
yes the string terminates with "\n", the placement of the triple quotes in last last has no impact in dedent. I think this was black formatterd.